users/postgres/clients.go

```diff
@@ -123,6 +123,36 @@ func (repo clientRepo) RetrieveByID(ctx context.Context, id string) (mgclients.C
 	return mgclients.Client{}, repoerr.ErrNotFound
 }
 
+func (repo clientRepo) RetrieveBasicInfoByID(ctx context.Context, id string) (mgclients.Client, error) {
+	q := `SELECT id, name, tags, metadata FROM clients WHERE id = :id`
```

Comment:
```suggestion
	q := `SELECT id, name FROM clients WHERE id = :id`
```

Reply:
Metadata and tags are not basic information since they can contain user information for example profile picture